### PR TITLE
Create InceptionContent and InceptionObject

### PR DESCRIPTION
**Pull request by leonfgama**
Create Content, InceptionObject, Show and Close Methods.

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -14,12 +14,17 @@
...
-      innerHTML: '<span>Hello world!</span>',
+    var myModal = inception.create({
+      innerHTML: '<h2 color="#FFF">Hello world!</h2>',
 	    selector: document.body,
-      overlayColor: '#f66',
+      overlayColor: '#000',
       opacity: 0.7
     });
+
+    console.log(myModal);
```
Comment (Mauricio32) [inline demo/index.html]: Retirar console.log inativos

File: demo/index.html [hunk 2]
```diff
@@ -14,12 +14,17 @@
...
+    //myModal.show();
```
Comment (Mauricio32) [inline demo/index.html]: Retirar comentários

**Review by Mauricio32 — CHANGES_REQUESTED** [review]

**Review by Mauricio32 — APPROVED** [review]
LGTM

**Review by rafael-lopes — APPROVED** [review]
LGTM

**Pull request merged**